core/os/android/adb/commands.go

```diff
@@ -406,15 +406,16 @@ func (b *binding) QueryAnglePackageName(ctx context.Context) (string, error) {
 	}
 	// ANGLE supported, so check for installed ANGLE package
 	// Favor custom installed package first, followed by default system package
-	custom := "com.chromium.angle"
+	//  org.chromium.angle is first because it's the most recent & preferred name
+	custom := [2]string{"org.chromium.angle", "com.chromium.angle"}
```

Comment:
Is ANGLE going to be `org.chromium.angle` moving forward? If yes, why can't everyone switch to it?

Reply:
Yes, I agree that just switching is cleaner going fwd. Updating to that.